saos-restful-web-api/src/main/java/pl/edu/icm/saos/api/services/exceptions/ControllersEntityExceptionHandler.java

```diff
@@ -62,6 +63,18 @@
         return createErrorResponse(builder, errorStatus);
     }
 
+    @ExceptionHandler(TypeMismatchException.class)
+    public ResponseEntity<Map<String, Object>> handleTypeMismatchError(TypeMismatchException ex){
+        ErrorReason errorStatus = ErrorReason.WRONG_REQUEST_PARAMETER_ERROR;
+        
+        ErrorRepresentation.Builder builder = create(errorStatus, ex);
+        builder.propertyName(ex.getPropertyName());
+        String message = String.format("parameter can't have value '%s'", ex.getValue());
```

Comment:
Could you give here a little more formal description, e.g.: incorrect parameter value '%s'
?